tests/api-testing/tests/test_foldersnew.py

```diff
@@ -0,0 +1,142 @@
+def retrieve_new_dashboard(session, base_url, org_id, folder_id):
```

Comment:
This test is retrieving the "dashboard folder" not the dashboard itself. You might want to update the name and and logs to reflect that in order to avoid confusion.